app/src/main/java/com/geeksville/mesh/ui/MapFragment.kt

```diff
@@ -1,680 +1,316 @@
-                            binding.tilePackText.visibility = View.INVISIBLE
-                            tileRegionDownloadSuccess = true
-
-                        } else {
-                            debug("Waiting for style pack download to be finished.")
-                        }
-                    }
-                }
-                expected.error?.let {
-                    tileRegionDownloadSuccess = false
-                    // Handle error occurred during the tile region download.
-                    binding.tilePackDownloadProgress.visibility = View.INVISIBLE
-                    binding.tilePackText.visibility = View.INVISIBLE
-                    debug("TileRegionError: $it")
-                }
-            }
-        } else {
-            Toast.makeText(
-                requireContext(),
-                R.string.download_region_connection_alert,
-                Toast.LENGTH_LONG
-            ).show()
-        }
+        map.onPause()
+        super.onPause()
     }
 
-    /**
-     * OnLongClick of the map set a position marker.
-     * If a user long-clicks again, the position of the first marker will be updated
-     */
-    private val longClick = OnMapLongClickListener {
-        val userDefinedPointImg =
-            ContextCompat.getDrawable(
-                requireActivity(),
-                R.drawable.baseline_location_on_white_24dp
-            )!!
-                .toBitmap()
-        point = Point.fromLngLat(it.longitude(), it.latitude())
-
-
-        /*
-        Calculate region from user specified position.
-        5 miles NE,NW,SE,SW from user center point.
-        25 Sq Mile Region
-        */
-        //____________________________________________________________________________________________
-        val topRight = calculateCoordinate(45.0, point?.latitude()!!, point?.longitude()!!)
-        val topLeft = calculateCoordinate(135.0, point?.latitude()!!, point?.longitude()!!)
-        val bottomLeft = calculateCoordinate(225.0, point?.latitude()!!, point?.longitude()!!)
-        val bottomRight = calculateCoordinate(315.0, point?.latitude()!!, point?.longitude()!!)
-        //____________________________________________________________________________________________
-
-        val pointList = listOf(topRight, topLeft, bottomLeft, bottomRight, topRight)
-
-        squareRegion = LineString.fromLngLats(pointList)
-
-        geoJsonSource = geoJsonSource(boundingBoxId) {
-            geometry(squareRegion)
-        }
-        lineLayer = lineLayer(lineLayerId, boundingBoxId) {
-            lineCap(LineCap.ROUND)
-            lineJoin(LineJoin.MITER)
-            lineOpacity(0.7)
-            lineWidth(1.5)
-            lineColor("#888")
-        }
-
-        if (point != null) {
-            binding.downloadRegion.visibility = View.VISIBLE
+    override fun onResume() {
+        super.onResume()
+        map.invalidate()
+        val tileSourceName = mPrefs.getString(
```

Comment:
I need to determine if this is actually needed after adding the setDestroyMode(false)

Reply:
Determined that this is not needed